FreeRTOS/FreeRTOS
# ARMv7-R No_GIC Port Demo

Description
-----------
FreeRTOS Port for the [ARMv7-R Architecture](https://developer.arm.com/documentation/ddi0406/cd/?lang=en), tested on the [Cortex R5](https://developer.arm.com/Processors/Cortex-R5) Processor.

Demo related to the [FreeRTOS-Kernel Pull Request](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1101) for ARMv7-R No_GIC Port.

The RM57 project's [CMakeLists.txt](https://github.com/kar-rahul-aws/FreeRTOS/blob/ARM_Crx_No_GIC_Demo/FreeRTOS/Demo/CORTEX_No_GIC_R5F_TI_RM57_HERCULES_GCC/CMakeLists.txt#L48) uses fetch content to checkout the correct tag of the repo

Both of these projects also have the necessary Code Composer Studio files and projects as well, which is mentioned in their READMEs.

Test Steps
-----------

The RM57 Demo was tested using the [LAUNCHXL2-RM57L](https://www.ti.com/tool/LAUNCHXL2-RM57L)
and the [TMDXRM57LHDK](https://www.ti.com/tool/TMDXRM57LHDK) Development Kits.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
